cfme/tests/cloud_infra_common/test_retirement_manual.py

```diff
@@ -0,0 +1,146 @@
+# -*- coding: utf-8 -*-
+# pylint: skip-file
+"""Manual tests"""
+import pytest
+
+from cfme import test_requirements
+
+pytestmark = [
+    pytest.mark.ignore_stream('5.10', 'upstream'),
+    pytest.mark.manual,
+    test_requirements.retirement
+]
+
+
+@pytest.mark.tier(2)
+def test_retire_infra_vms_folder():
+    """
+    test the retire funtion of vm on infra providers, at least two vm,
+    retire now button vms page
+
+    Polarion:
+        assignee: tpapaioa
+        casecomponent: Provisioning
+        caseimportance: medium
+        initialEstimate: 1/2h
+    """
+    pass
+
+
+def test_retirement_date_uses_correct_time_zone():
+    """
+    Bug 1565128 - Wrong timezone when selecting retirement time
+
+    Bugzilla:
+        1565128
+
+    After saving VM retirement date/time (using both "Specific Date and
+    Time" and "Time Delay from Now" options), the displayed Retirement
+    Date has the correct date and time-zone appropriate time.
+
+    Polarion:
+        assignee: tpapaioa
+        casecomponent: Infra
+        caseimportance: medium
+        initialEstimate: 1/15h
+        startsin: 5.9
```

Comment:
**Question** Was this available in 5.9? If so, then it should be in 5.10 correct?